packages/eslint-plugin/src/configs/base.js

```diff
@@ -19,10 +19,9 @@ module.exports = {
   },
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33448&commitId=7f62a0148d259bb7c557bf6524bdf7a3298cf713&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|9| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>